docs/pilot_documentation/aws_pilot.rst

```diff
@@ -1,4 +1,26 @@
 AWS Pilot
 ================
 
-Short description of the AWS pilot
+The AWS Pilot is used to work with the AWS API.
+Supports local execution.
+
+Main Languages
+^^^^^^^^^^^^^^^^^^^^
+
+* Braket
+* QASM 3
+
+**Note:** The transpile manager allows for the use of any language supported by the transpile manager.
+
+Supported Job Types
+^^^^^^^^^^^^^^^^^^^^
+
+Runner
+*******
+
+**Description:** Executed a simple Job on the AWS API.
```

Comment:
es wird hier ja nur lokal gemacht und kein api call

Reply:
also klar steht es in der "Note" aber einfach weglassen, weil es aktuell falsch ist oder direkt ein vermerk, dass es theoretisch noch kommen soll